app/server.go

```diff
@@ -617,6 +623,18 @@ func (s *Server) initLogging() error {
 	return nil
 }
 
+func (s *Server) enableLoggingMetrics() {
+	if s.Metrics == nil {
+		return
```

Comment:
This is an expected path for TE, right? That's why we don't need to log it?

Reply:
Correct